partcad/src/partcad/lint/python.py

```
@@ -0,0 +1,48 @@
+from partcad.lint.lint import Linting, LintingReport, Severity
+
+
+class PythonLinting(Linting):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+
+    def get_targets(self, ctx: Context, package: Project) -> list[str]:
+        """Return a list of Python files in the package's config directory."""
+        return [os.path.join(package.config_dir, f) for f in os.listdir(package.config_dir) if f.endswith(".py")]
+
+    async def validate(self, ctx: Context, package: Project, target: str, lint_ctx: dict = {}) -> LintingReport:
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)